### PR TITLE
Automated playground deployment trigger for new lakeFS release

### PR DESCRIPTION
**Pull request by idanovo**

### CODE REVIEW

File: .github/workflows/docker-publish.yaml [hunk 1]
```diff
@@ -90,3 +90,21 @@ jobs:
...
+          https://api.github.com/repos/${{ secrets.REPOSITORY_NAME }}/actions/workflows/${{ secrets.WORKFLOW_ID_STAGING }}/dispatches \
+          --data '{"ref":"main", "inputs": { "lakefs_image": ${{ steps.version.outputs.tag }} }}'
+
+          curl \
+          -X POST \
+          -H "Authorization: Bearer ${{ secrets.PAT_TOKEN }}"\
+          -H "Accept: application/vnd.github+json" \
+          -H "Content-Type: application/json" \
+          https://api.github.com/repos/${{ secrets.REPOSITORY_NAME }}/actions/workflows/${{ secrets.WORKFLOW_ID_PRODUCTION }}/dispatches \
+          --data '{"ref":"main", "inputs": { "lakefs_image": ${{ steps.version.outputs.tag }} }}'
```
Comment (ortz) [inline .github/workflows/docker-publish.yaml:110]: @idanovo this doesn't allow you to create a dependency between the workflows (and test the outcome of the change).

Comment (idanovo) [reply .github/workflows/docker-publish.yaml]: @ortz you are right. changed to one one job with dependency

**Review by ortz — APPROVED** [review]

**Review by Jonathan-Rosenberg — CHANGES_REQUESTED** [review]
shouldn't you use [Reusable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows)?

**Comment by idanovo** [comment]
> shouldn't you use [Reusable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows)?

Looks cool, but I'm not sure how it's related.
I'm triggering here workflow of another repository.

**Comment by Jonathan-Rosenberg** [comment]
@idanovo oh miss that detail...
Thanks!

**Review by Jonathan-Rosenberg — APPROVED** [review]

**Pull request merged**